### PR TITLE
Update protobuf

### PR DESCRIPTION
**Pull request by jobevers**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -8,11 +8,11 @@ ecdsa==0.13
 gmpy==1.17
 jsonrpc==1.2
 jsonrpclib==0.1.7
-https://github.com/lbryio/lbryum/tarball/master/#egg=lbryum
+https://github.com/lbryio/lbryum/tarball/update-protobuf/#egg=lbryum
```
Comment (jackrobison) [inline requirements.txt]: should this be master? or should that be a separate commit after lbryum gets merged?

Comment (jobevers) [inline requirements.txt]: yes, I'll revert this commit after lbryum gets merged.  I needed to
reference that branch to test the build process in the meantime.
